README.md

```diff
@@ -1 +1,35 @@
-# dotaKeep
\ No newline at end of file
+mobile
+====
+
+Overview
```

Comment:
I'd remove the sections about OpenDota core, since it might be kind of confusing.

This includes the "raw data", "public deployment", blog post links, and the "started in august 2014" bits.